src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -29,25 +37,80 @@ protected function configure()
      * @return void
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
```

Comment:
Please update the return type here to match its parent class. Console Commands should return `int|null` and add proper return. Please return 0 or 1 not null.

Reply:
Please fix this also for the other commands.